src/components/10-atoms/radio/index.scss

```diff
@@ -166,18 +166,19 @@ axa-radio {
       height: auto;
```

Comment:
I think the `padding: inherit` could be related to this line. So, "inherit" for the padding below could make sense.

Reply:
`padding: inherit` would never inherit something from the `height` property. `padding-top` from `a-radio__content` inherits from `padding` from the class `.a-radio__wrapper`. This creates a dependency that shouldn't be. We definitely have to remove `inherit`.

Reply:
What i mean: if height is variable it could have a side-effect to the padding. But if i think about it... that possible wrong. So .. you are doing right:)